src/patternfly/components/Switch/switch-input.hbs

```diff
@@ -1,7 +1,7 @@
 <input class="pf-c-switch__input{{#if switch-input--modifier}} {{switch-input--modifier}}{{/if}}"
   type="checkbox"
-  id="{{id}}"
-  aria-labelledby="switch-is-{{id}}"
+  id="switch-is-{{id}}"
+  aria-label="switch-is-{{id}}"
```

Comment:
Hmm, the switch currently has other issues, which should be addressed as part of 
https://github.com/patternfly/patternfly-next/pull/1473. 

Since this issue should also be addressed as part of that PR, then I would pull these changes out.

Reply:
Thanks for pointing out #1473, @jgiardino! Should we put the aria switch component changes aside for now? It's a big chunk of the violation currently.